### ISSUE TITLE
Feature Request : Pull Requests to Specific Branch

### ISSUE THREAD

**Issue by DamianCasale** [nishio-dens/bitbucket-pullrequest-builder-plugin#61]
I would like the ability to have a build triggered only if the pull request was directed to a specific branch.
Maybe I have overlooked some functionality in jenkins or this plugin and this is already possible else this addition would be great to allow overview of different parts of our project.

In Jenkins I would like a project for each important branch in our code base. For example....

```
 * project (master)
 * project (alpha)
 * project (release 1.5)
```

So one for main line work in master (where developers will add pull requests from their feature branches), one for alpha and one for previous releases (for fixes).

At the moment with a basic set-up of this plugin: If I do a pull request to any branch we will get a build triggered on all 3 projects, worst still if the build fails and a comment is added to the pull request by this plugin, it re-triggers builds on the other two projects causing an infinite loop of builds.

**Comment by DamianCasale** [comment]
This could also maybe be extended for other parameters like pull request must have *n* approvals before it is built.

**Comment by maxvodo** [comment]
Try my fork https://github.com/maxvodo/bitbucket-pullrequest-builder-plugin with partial support this feature

**Comment by maxvodo** [comment]
@DamianCasale Ready for production @96ab7a75f14d9990f3c8f1255f9790c496a64473

**Comment by maxvodo** [comment]
@nishio-dens, you may close this issue (feature available after merging pr #71).
